src/main.module.ts

```diff
@@ -32,7 +32,7 @@ import {
           transport: {
             target: 'pino-pretty',
             options: {
-              singleLine: true,
+              singleLine: config.get<string>('app.singleLineLogFormat'),
```

Comment:
Can we use `config.get<boolean>` please?

Reply:
updated